src/routers/upload.js

```diff
@@ -0,0 +1,20 @@
+const { Router } = require('express');
+const uploadHandler = require('../middleware/uploadHandler');
+const importationService = require('../services/importationService');
+
+const router = new Router();
+
+router.post('/upload', uploadHandler('local'), (req, res) => {
+  if (req.file) {
+    const fileFolder = req.file.destination.split('/');
+    const uuid = fileFolder[fileFolder.length - 1];
+    return importationService.createImportation(uuid, req.file.destination, 'offers', 123)
+      .then((result) => {
+        console.log(result);
+        res.send('Thank you for the file');
+      });
+  }
+  return res.end('Missing file');
```

Comment:
instead of this other return, this should be verified at validation phase, dont you think?